sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -860,12 +860,23 @@ def _add_argparse_args(cls, parser):
             'https://cloud.google.com/compute/docs/vpc/'))
     parser.add_argument(
         '--worker_harness_container_image',
+        dest='sdk_container_image',
```

Comment:
Would not this be backward incompatible and break usage like: 
```options.view_as(WorkerOptions).worker_harness_container_image``` ?

Reply:
hmm, I talked to @tvalentyn and he doesn't think this should be a concern for users since the flag is mostly used for official Dataflow images only/in preview, but we can make it backwards-compatible.

Reply:
+1 for making it fully backwards compatible just in case.